recipes/civetweb/all/conanfile.py

```diff
@@ -82,20 +82,31 @@ def package(self):
                 os.remove(os.path.join(bin_folder, bin_file))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.components["_civetweb"].names["cmake_find_package"] = "civetweb"
+        self.cpp_info.components["_civetweb"].names["cmake_find_package_multi"] = "civetweb"
+        self.cpp_info.components["_civetweb"].names["pkg_config"] = "civetweb"
```

Comment:
Is there an official pkg-config file? if not, I would highly recommend to not use this value for this component, because it overrides the global pkg-config file generated by conan for this recipe (@ericLemanissier will confirm).